docs/src/physics/turbulence_closures.md

```diff
@@ -25,9 +25,9 @@ Each tracer may have a unique diffusivity ``\kappa``.
 
 ## Constant anisotropic diffusivity
 
-In Oceananigans.jl, a constant anisotropic diffusivity implies a constant tensor
+A constant anisotropic diffusivity implies a constant tensor
 diffusivity ``\nu_{j k}`` and stress ``\boldsymbol{\tau}_{ij} = \nu_{j k} u_{i, k}`` with non-zero
-components ``\nu_{11} = \nu_{22} = \nu_h`` and ``\nu_{33} = \nu_v``.
+components ``\nu_{11} = \nu_{22} = \nu_h`` and ``\nu_{33} = \nu_z``.
```

Comment:
This section is outdated right? There's no anisotropic diffusivity anymore.

Reply:
Yes, it's a bit outdated...